docs/user_guide/web-components.rst

```diff
@@ -4,16 +4,31 @@
 Sphinx Design Components
 ========================
 
-The PyData Sphinx Theme uses `sphinx-design <https://sphinx-design.readthedocs.io/en/latest/index.html>`__
-to add several UI components and provide extra flexibility for content creation.
-These include badges, buttons, cards, and tabs, among other components.
-This theme provides custom CSS to ensure that `sphinx-design <https://sphinx-design.readthedocs.io/en/latest/index.html>`__ elements look and feel consistent with this theme.
+On this page, you will find user interface components such as badges, buttons,
```

Comment:
Updating this page in the docs made me realize that this part of the docs could really use a restructure:

- This page is titled Sphinx Design but it also includes info about two other Sphinx extensions, sphinx-copybutton and sphinx-togglebutton, that have nothing to do with sphinx-design.
- More confusingly, the next page in the docs, "Extending the theme" has a section about sphinx-togglebutton, as if that extension wasn't just discussed on the previous page.
- This page is under: "User Guide / Content and features." The "Content and features" section has four pages, three of which are about other extensions that our theme is designed to work with. This makes me think that we should probably have a dedicated section on such extensions. And each page in that section should be dedicated to that particular extension (even if it means that some of those pages will be really short).

Reply:
I agree that these sections could be restructured. I will open a separate issue for this, and we can action it from there.